src/puzzles/puzzle.gd

```diff
@@ -9,5 +9,5 @@ func reset() -> void:
 	push_error("Not implemented")
 
 
-func load_puzzle(puzzle_to_load: PackedScene) -> void:
+func load_puzzle(_puzzle_to_load: PackedScene) -> void:
```

Comment:
While you're at it, can you add a `@warning_ignore` to the top on line 5 so it stops complaining about the signal not being used :stuck_out_tongue: There is an "[Ignore]" button in the warnings panel which will add this line for you